packages/gatsby/src/redux/actions/public.js

```diff
@@ -379,6 +379,7 @@ ${reservedFields.map(f => `  * "${f}"`).join(`\n`)}
       !fileContent.includes(`module.exports`) &&
       !fileContent.includes(`exports.default`) &&
       !fileContent.includes(`exports["default"]`) &&
+      !fileContent.match(/export \{.* as default.*\}/s) &&
```

Comment:
```suggestion
      !fileContent.match(/export\s*\{.*\s+as\s+default.*\}/s) &&
```
Not sure if this suggestion is totally correct, but I think this should match a bit more loose when it comes to whitespaces

so examples like:
```
export{X as default}
export     {X      as      default}
```
are also valid

Reply:
(that said, I realize our current code - specifically one for `export default` doesn't handle this nicely)

Reply:
Yeah I figured this was more or less true of almost all the checks and was trying to keep the regex simple.
Do you have any thoughts on the even simpler `!fileContent.includes('as default')`?

Reply:
I think we can start with your current code (and not apply my suggestion) and think about making all of those tests to be more resilient when we will get reports of problems

Reply:
sgtm, thanks :)